sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathExpressionsSuite.scala

```diff
@@ -691,6 +803,19 @@ class MathExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
       checkEvaluation(BRound(Literal.create(null, dataType), Literal(2)), null)
       checkEvaluation(BRound(Literal.create(null, dataType),
         Literal.create(null, IntegerType)), null)
+      checkEvaluation(checkDataTypeAndCast(
+        RoundFloor(Literal.create(null, dataType), Literal(2))), null)
+      try {
+        checkEvaluation(checkDataTypeAndCast(RoundFloor(Literal.create(null, dataType),
+          Literal.create(null, IntegerType))), null)
+        checkEvaluation(checkDataTypeAndCast(RoundCeil(Literal.create(null, dataType),
```

Comment:
it looks weird to test the error behavior in this test case. Can we move it to `ceil-floor-with-scale-param.sql`?